### ISSUE TITLE
The issue for the 'git log --follow' in file history feature

### ISSUE THREAD

**Issue by leolcao** [huizhougit/githd#92]
From beginning, I found the output of file history is not follow the renamed case for one file, so not as the same output of `git log --follow <path/to/file>`, I take it as missing feature of this extension, but once I dive into the source codes, I found it already considered this case, but unfortunately, contains a bug from below lines in the gitService.ts

```ts
async getLogEntries(
...
if (file) {
        const filePath = (await this.getGitRelativePath(file)) ?? '.';
        if (line) {
          args.push(`-L ${line},${line}:${filePath}`, '--');
        } else {
          args.push('--', '--follow', filePath);
        }
      } else {
        // the '--' is to avoid same branch and file names caused  error
        args.push('--');
      }
...
```

The bug comes from line: `args.push('--', '--follow', filePath);`, it should be `args.push('--follow', filePath);`

from the verbose logs, I verified my thinking, and I manually modified the installed file `huizhou.githd-2.5.4/dist/extension.js`, 

![image](https://github.com/user-attachments/assets/a17be2ec-b318-45c7-a92e-d0560a0705fe)

then, it works to get all output from the argument `--follow`, and the verbose logs show correct info:

```shell
[09:36:45.168][Verbose] git command: git log --skip=0 --max-count=90 --date-order --simplify-merges main --format=[githd-es]%s[githd-fs]%h[githd-fs]%d[githd-fs]%aN[githd-fs]%ae[githd-fs]%ct[githd-fs]%cd[githd-fs]%cr[githd-fs] --date=local --follow 1.cpp/1.utility/rxcpp-vcpkg/Nuget.config. Output size: 6767 (5943ms) e:\llc\techstack.git\
```

the incorrent log:

```shell
[09:36:45.168][Verbose] git command: git log --skip=0 --max-count=90 --date-order --simplify-merges main --format=[githd-es]%s[githd-fs]%h[githd-fs]%d[githd-fs]%aN[githd-fs]%ae[githd-fs]%ct[githd-fs]%cd[githd-fs]%cr[githd-fs] --date=local -- --follow 1.cpp/1.utility/rxcpp-vcpkg/Nuget.config. Output size: 6767 (5943ms) e:\llc\techstack.git\
```

**Comment by huizhougit** [comment]
@leolcao thanks for reporting and looking into the issue. It should be fixed by https://github.com/huizhougit/githd/commit/35bafa35c5fe8fdb91dcb0b9bdc75e3114204a76

**Comment by leolcao** [comment]
@huizhougit verified this bug is fixed from version 2.5.5, thanks for the quick response and update, and with best wishes~
